.github/actions/test/action.yaml

```diff
@@ -7,5 +7,5 @@ runs:
     run: cargo test --manifest-path tools/slicec-cs/Cargo.toml
     shell: bash
   - name: 🧪 Test
-    run: dotnet test --no-build --verbosity normal
+    run: dotnet test --no-build --verbosity normal --blame-hang-timeout 8m
```

Comment:
`--blame-hang-timeout` is used to terminate the tests and dump core after the timeout, the NUnit timeout attribute is now deprecated.